### PR TITLE
Add table googledirectory_domain_alias. Closes #8

### LINKED ISSUE #8: Add table googledirectory_domain_alias
**References**
https://developers.google.com/admin-sdk/directory/reference/rest/v1/domainAliases
https://registry.terraform.io/providers/hashicorp/googleworkspace/latest/docs/resources/domain_alias

### PR DESCRIPTION
**Pull request by bigdatasourav**
# Example query results
<details>
  <summary>Results</summary>
  
```
> select * from googledirectory_domain_alias where domain_alias_name = 'turbot-dev.com.test-google-a.com'
+----------------------------------+---------------------+----------+-------------+---------------------------------------------------------------------------+-----------------------------+---------------
| domain_alias_name                | creation_time       | verified | customer_id | etag                                                                      | kind                        | parent_domain_
+----------------------------------+---------------------+----------+-------------+---------------------------------------------------------------------------+-----------------------------+---------------
| turbot-dev.com.test-google-a.com | 2021-07-06 21:15:36 | true     | <null>      | "xleqcsVBDtt63aQahfMIqXdQqgrChHYB52dVIG_3_14/cMhgpt9sMgIdptf7osVqVEa3nD0" | admin#directory#domainAlias | turbot-dev.com
+----------------------------------+---------------------+----------+-------------+---------------------------------------------------------------------------+-----------------------------+---------------
> select
  domain_alias_name,
  creation_time,
  verified
from
  googledirectory_domain_alias;
+----------------------------------+---------------------+----------+
| domain_alias_name                | creation_time       | verified |
+----------------------------------+---------------------+----------+
| turbot-dev.com.test-google-a.com | 2021-07-06 21:15:36 | true     |
+----------------------------------+---------------------+----------+
> select
  domain_alias_name,
  creation_time,
  verified
from
  googledirectory_domain_alias
where
  not verified;
+-------------------+---------------+----------+
| domain_alias_name | creation_time | verified |
+-------------------+---------------+----------+
+-------------------+---------------+----------+

```
</details>

### CODE REVIEW

File: googledirectory/table_googledirectory_domain_alias.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableGoogleDirectoryDomainAlias(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "googledirectory_domain_alias",
+		Description: "Domain Alias defined in the Google Workspace directory.",
```
Comment (Subhajit97) [inline googledirectory/table_googledirectory_domain_alias.go:16]:
```suggestion
		Description: "Domain alias defined in the Google Workspace directory.",
```

File: googledirectory/table_googledirectory_domain_alias.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "kind",
+				Description: "The type of the API resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "parent_domain_name",
+				Description: "The parent domain name that the domain alias is associated with.",
```
Comment (Subhajit97) [inline googledirectory/table_googledirectory_domain_alias.go:65]: Move this column after `domain_alias_name`

File: googledirectory/table_googledirectory_domain_alias.go [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+		return nil, err
+	}
+
+	// Set default value to my_customer, to represent current account
+	customerID := "my_customer"
+	if d.KeyColumnQuals["customer_id"] != nil {
+		customerID = d.KeyColumnQuals["customer_id"].GetStringValue()
+	}
+
+	resp, err := service.DomainAliases.List(customerID).Do()
```
Comment (Subhajit97) [inline googledirectory/table_googledirectory_domain_alias.go:87]: Support filtering using `parent_domain_name`

File: docs/tables/googledirectory_domain_alias.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+```sql
+select
+  domain_alias_name,
+  creation_time,
+  verified
+from
+  googledirectory_domain_alias
+where
+  not verified;
+```
```
Comment (Subhajit97) [inline docs/tables/googledirectory_domain_alias.md:29]: Add example to list domain alias by parent domain

**Review by Subhajit97 — APPROVED** [review]

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**